ManagementLogger attempts to lookup TinkerPop's GraphManager

This can be recreated by running a [basic Java example](https://github.com/pluradj/janusgraph-java-example/) that does not include a [dependency](https://github.com/pluradj/janusgraph-java-example/blob/master/pom.xml#L27-L31) for `org.apache.tinkerpop:gremlin-server`.

```
java.lang.NoClassDefFoundError: org/apache/tinkerpop/gremlin/server/GraphManager
    at java.lang.ClassLoader.defineClass1 (Native Method)
    at java.lang.ClassLoader.defineClass (ClassLoader.java:763)
    at java.security.SecureClassLoader.defineClass (SecureClassLoader.java:142)
    at java.net.URLClassLoader.defineClass (URLClassLoader.java:467)
    at java.net.URLClassLoader.access$100 (URLClassLoader.java:73)
    at java.net.URLClassLoader$1.run (URLClassLoader.java:368)
    at java.net.URLClassLoader$1.run (URLClassLoader.java:362)
    at java.security.AccessController.doPrivileged (Native Method)
    at java.net.URLClassLoader.findClass (URLClassLoader.java:361)
    at java.lang.ClassLoader.loadClass (ClassLoader.java:424)
    at java.lang.ClassLoader.loadClass (ClassLoader.java:357)
    at org.janusgraph.graphdb.database.management.ManagementLogger$SendAckOnTxClose.run (ManagementLogger.java:240)
    at java.lang.Thread.run (Thread.java:748)
Caused by: java.lang.ClassNotFoundException: org.apache.tinkerpop.gremlin.server.GraphManager
    at java.net.URLClassLoader.findClass (URLClassLoader.java:381)
    at java.lang.ClassLoader.loadClass (ClassLoader.java:424)
    at java.lang.ClassLoader.loadClass (ClassLoader.java:357)
    at java.lang.ClassLoader.defineClass1 (Native Method)
    at java.lang.ClassLoader.defineClass (ClassLoader.java:763)
    at java.security.SecureClassLoader.defineClass (SecureClassLoader.java:142)
    at java.net.URLClassLoader.defineClass (URLClassLoader.java:467)
    at java.net.URLClassLoader.access$100 (URLClassLoader.java:73)
    at java.net.URLClassLoader$1.run (URLClassLoader.java:368)
    at java.net.URLClassLoader$1.run (URLClassLoader.java:362)
    at java.security.AccessController.doPrivileged (Native Method)
    at java.net.URLClassLoader.findClass (URLClassLoader.java:361)
    at java.lang.ClassLoader.loadClass (ClassLoader.java:424)
    at java.lang.ClassLoader.loadClass (ClassLoader.java:357)
    at org.janusgraph.graphdb.database.management.ManagementLogger$SendAckOnTxClose.run (ManagementLogger.java:240)
    at java.lang.Thread.run (Thread.java:748)
```

`gremlin-server` is listed as an optional dependency in [`janusgraph-core`](https://github.com/JanusGraph/janusgraph/blob/v0.3.0/janusgraph-core/pom.xml#L44), so I wouldn't expect to see this `ClassNotFoundException` even if the dependency is unavailable.

The examples in the JanusGraph repo don't run into this issue because they include a dependency on [`janusgraph-test`](https://github.com/JanusGraph/janusgraph/blob/v0.3.0/janusgraph-test/pom.xml#L56) which pulls in `gremlin-server`.

--- a/janusgraph-core/src/main/java/org/janusgraph/graphdb/database/management/ManagementLogger.java
+++ b/janusgraph-core/src/main/java/org/janusgraph/graphdb/database/management/ManagementLogger.java
@@ -16,6 +16,7 @@ package org.janusgraph.graphdb.database.management;
 
 import com.google.common.base.Preconditions;
 import com.google.common.collect.ImmutableSet;
+import org.janusgraph.core.JanusGraphManagerUtility;
 import org.janusgraph.core.JanusGraphTransaction;
 import org.janusgraph.core.schema.JanusGraphManagement;
 import org.janusgraph.diskstorage.ResourceUnavailableException;
@@ -237,7 +238,7 @@ public class ManagementLogger implements MessageReader {
                         txStillOpen = true;
                     }
                 }
-                final JanusGraphManager jgm = JanusGraphManager.getInstance();
+                final JanusGraphManager jgm = JanusGraphManagerUtility.getInstance();
                 final boolean janusGraphManagerIsInBadState = null == jgm && action.equals(EVICT);
                 if (!txStillOpen && janusGraphManagerIsInBadState) {
                     log.error("JanusGraphManager should be instantiated on this server, but it is not. " +

